server/issue_parser.go

```diff
@@ -32,14 +32,20 @@ func reporterSummary(reporter *jira.User) string {
 	return reporterSummary
 }
 
-func getActions(instanceID types.ID, client Client, issue *jira.Issue) ([]*model.PostAction, error) {
+func getActions(p *Plugin, instanceID types.ID, client Client, issue *jira.Issue) ([]*model.PostAction, error) {
 	var actions []*model.PostAction
 
 	ctx := map[string]interface{}{
 		"issue_key":   issue.ID,
```

Comment:
Strange that the map key is called `issue_key`, which I assume is the "MM-1000" format, but what is being assigned is a numeric id.